README.md

```diff
@@ -1,185 +1,164 @@
-# Installing FUN CMS
+# Richie, A FUN CMS for Open edX
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/openfun/richie.svg)](https://greenkeeper.io/)
 
 [![CircleCI](https://circleci.com/gh/openfun/richie/tree/master.svg?style=svg)](https://circleci.com/gh/openfun/richie/tree/master)
 
-This document aims to list all needed steps to have a working `FUN CMS` installation on your laptop.
+## Overview
 
-A better approach is to use [`Docker`](https://docs.docker.com). To get started, you can read [explanations](docs/docker.md) on how `FUN CMS` uses Docker for development.
+`Open edX` is a great tool for authoring (**Studio**), hosting and playing (**LMS**) interactive
+online courses and MOOCs.
 
+However, if you need to build a complete website with flexible content to aggregate your courses,
+in several languages and from different sources, **you will soon need a CMS**.
 
-## Installing a fresh server
+For example, the [edx.org](https://www.edx.org) website is built on `Drupal` for content pages and
+on [edX course discovery](https://github.com/edx/course-discovery) for the catalog.
 
-### Version
+At `France Université Numérique`, we wanted to build a similar portal but with `Python` and
+`Django`. That's why we built `Richie` on top of [DjangoCMS](https://www.django-cms.org), one of
+the best CMS on the market, as a toolbox to easily create full fledged websites with a catalog of
+online courses.
 
-You need a `Ubuntu 16.04 Xenial Xerus` (the latest LTS version) fresh installation.
+Among the functionalities that `Richie` offers out of the box:
 
-If you are using another operating system or distribution, you can use [`Vagrant`](https://docs.vagrantup.com/v2/getting-started/index.html) to get a running Ubuntu 16.04 server in seconds.
+- multi-site and multi-lingual by default,
+- advanced access rights and moderation,
+- catalog of courses synchronized with one or more `Open edX` instances,
+- search engine based on `Elasticsearch` and pre-configured to offer full-text queries,
+  multi-facetting, auto-complete,...
+- flexible custom pages for courses, organizations, subjects, teachers (and their inter-relations),
+- Extensible with any third-party `DjangoCMS` plugin or any third-party `Django` application.
 
```

Comment:
This whole section should stand in a "motivations" section from the project's documentation, not in a README. IMO a README should go straight to the point: one phrase to describe what purpose serves the project, a short section explaining how to install & test it, contributing, license and voilà.